packages/models/src/lib/notifications/publish-status.ts

```diff
@@ -0,0 +1,5 @@
+export enum PublishStatus {
```

Comment:
This has to be a numeric enum, because if we use strings, the enum won't work in a capped collection. They impose a restriction that document sizes can't change, and unless all our enum variants are the exact same string length, changing the enum value will change the document size.